westcoast/src/main/java/frc/robot/Robot.java

```diff
@@ -79,10 +91,22 @@ public void autonomousExit() {
 
     @Override
     public void teleopInit() {
-        Commands
-                .TeleopCommands
-                .raw(definition)
-                .schedule();
+        switch (driveModeChooser.getSelected()) {
```

Comment:
Why use an enum when you can just send the variable to the dashboard and then schedule the dashboard's selected? In theory that only recreates the command once so it shouldn't cause any issues. This would be similar to the autonchooser

Reply:
yeah went back and forth on that. I may change before merge. lets see